src/webclients/components/sdfv/utils/attributes_table.ts

```diff
@@ -137,6 +137,118 @@ export function attrTablePutCode(
     );
 }
 
+export function attrTablePutData(
+    key: string, subkey: string | undefined, val: any, elem: any | undefined,
+    xform: any | undefined, target: any, cell: JQuery, dtype: string,
+    meta: any, editableKeys: boolean = false
+): DictProperty {
+    const dataCellContainer = $('<div>', {
+        'class': 'popup-editable-property-container',
+    }).appendTo(cell);
+    //$('<div>', {
+    //    'html': `
+    //        <span title="${val /*sdfg_property_to_string(
+    //            val, VSCodeRenderer.getInstance()?.view_settings()
+    //        )*/}">Details</span>
+    //    `,
+    //}).appendTo(dataCellContainer);
```

Comment:
what's this

Reply:
Stale, removed. Wanted to originally have a 'Details' text next to the edit button that shows a tooltip with all the properties. This proved to be too noisy UI wise, so I removed.